var/spack/repos/builtin/packages/speexdsp/package.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Speexdsp(AutotoolsPackage):
+    """SpeexDSP is a patent-free, Open Source/Free Software DSP library."""
+
+    homepage = "https://github.com/xiph/speexdsp"
+    url      = "https://github.com/xiph/speexdsp/archive/SpeexDSP-1.2.0.tar.gz"
+
+    version('1.2.0', sha256='d7032f607e8913c019b190c2bccc36ea73fc36718ee38b5cdfc4e4c0a04ce9a4')
+
+    depends_on('autoconf', type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool',  type='build')
+    depends_on('m4',       type='build')
+
+    depends_on('fftw-api')
+
+    patch('mkl.patch')
+
+    def autoreconf(self, spec, prefix):
+        autoreconf('--install', '--verbose', '--force')
```

Comment:
I think this is the default, you should be able to remove this.

Reply:
It does not seem that way:
```console
==> speexdsp: Executing phase: 'autoreconf'
==> Error: ProcessError: Command exited with status 1:
    '/opt/packages/gpjohnsn/spack/opt/spack/linux-centos7-x86_64/gcc-8.4.0/autoconf-2.69-lmt2t6fktjdnzfra2vbwd5i5a4bdkpvu/bin/autoreconf' '-ivf' '-I' '/opt/packages/gpjohnsn/spack/opt/spack/linux-centos7-x86_64/gcc-8.4.0/automake-1.16.2-5d3hhw3swfo7irgn3t32rrvb4uzcauqv/share/aclocal' '-I' '/opt/packages/gpjohnsn/spack/opt/spack/linux-centos7-x86_64/gcc-8.4.0/libtool-2.4.6-i7gjyelqx73x44b7dhbrrfo2rzfudgay/share/aclocal'

1 error found in build log:
     5     ==> [2020-10-19-15:57:29.049121] Warning: *        a custom AUTORECONF phase in the package       *
     6     ==> [2020-10-19-15:57:29.049153] Warning: *********************************************************
     7     ==> [2020-10-19-15:57:29.050305] '/opt/packages/gpjohnsn/spack/opt/spack/linux-centos7-x86_64/gcc-8.4.0/autoconf-2.69-lmt2t6fktjdnzfra2
           vbwd5i5a4bdkpvu/bin/autoreconf' '-ivf' '-I' '/opt/packages/gpjohnsn/spack/opt/spack/linux-centos7-x86_64/gcc-8.4.0/automake-1.16.2-5d3h
           hw3swfo7irgn3t32rrvb4uzcauqv/share/aclocal' '-I' '/opt/packages/gpjohnsn/spack/opt/spack/linux-centos7-x86_64/gcc-8.4.0/libtool-2.4.6-i
           7gjyelqx73x44b7dhbrrfo2rzfudgay/share/aclocal'
     8     autoreconf: Entering directory `.'
     9     autoreconf: configure.ac: not using Gettext
     10    autoreconf: running: aclocal -I /opt/packages/gpjohnsn/spack/opt/spack/linux-centos7-x86_64/gcc-8.4.0/automake-1.16.2-5d3hhw3swfo7irgn3
           t32rrvb4uzcauqv/share/aclocal -I /opt/packages/gpjohnsn/spack/opt/spack/linux-centos7-x86_64/gcc-8.4.0/libtool-2.4.6-i7gjyelqx73x44b7dh
           brrfo2rzfudgay/share/aclocal --force -I m4
  >> 11    aclocal: error: couldn't open directory 'm4': No such file or directory
     12    autoreconf: aclocal failed with exit status: 1
```